dblp-aliases.csv

```diff
@@ -993,6 +993,7 @@ Gregory F. Welch,Greg Welch
 Nils Gregor Leander,Gregor Leander
 Greg Butler,Gregory Butler
 Gregoris Mentzas,Gregory Mentzas
+Gregory Bodwin,Greg Bodwin,Gregory Michael Bodwin

```

Comment:
You need one pair of alias/name pairs per line.

Reply:
Thank you. I just fixed it.